app/scripts/controllers/overview.js

```diff
@@ -418,10 +417,6 @@ angular.module('openshiftConsole')
       _.each(dcNames, function(dcName) {
         $scope.recentPipelinesByDC[dcName] = $scope.recentPipelinesByDC[dcName] || [];
         $scope.recentPipelinesByDC[dcName].push(build);
-        if (isIncompleteBuild(build)) {
-          $scope.incompletePipelinesByDC[dcName] = $scope.incompletePipelinesByDC[dcName] || [];
```

Comment:
@jwforres This scope var is no longer used. (Should have removed it as part of #1124)